nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -3185,7 +3187,7 @@ int nxagentCheckWindowConfiguration(XConfigureEvent* X)
     #endif
   }
 
-  if (result && nchildren_return)
+  if (children_return)
```

Comment:
Good! While it may look like `result && nchildren_return` are equivalent to checking for `children_return` to not be`NULL`, there is a corner case in which `nchildren_return` could be `0`, but `children_return` would still have been allocated a `sizeof(Window)` memory block.